moveit_demo_nodes/run_move_group/launch/run_move_group.launch.py

```diff
@@ -46,7 +50,7 @@ def generate_launch_description():
     ompl_planning_pipeline_config['move_group'].update(ompl_planning_yaml)
 
     # Trajectory Execution Functionality
-    controllers_yaml = load_yaml('run_move_group', 'config/controllers.yaml')
+    controllers_yaml = load_yaml('moveit_resources_panda_moveit_config', 'config/panda_controllers.yaml')
```

Comment:
If this is something MoveIt related, why not call it `panda_moveit_controllers.yaml`? People could misunderstand this as something ros2_control related. (This is maybe just my `ros2_control` background, I don't know.....)

Reply:
I agree with you, I'll rename it to `panda_moveit_simple_controllers.yaml`